ctgan/synthesizers/tvae.py

```diff
@@ -176,6 +188,26 @@ def fit(self, train_data, discrete_columns=()):
                 optimizerAE.step()
                 self.decoder.sigma.data.clamp_(0.01, 1.0)
 
+                batch.append(id_)
+                loss_values.append(loss.detach().cpu().item())
+
+            epoch_loss_df = pd.DataFrame({
+                'Epoch': [i] * len(batch),
+                'Batch': batch,
+                'Loss': loss_values
+            })
+            if not self.loss_values.empty:
+                self.loss_values = pd.concat(
+                    [self.loss_values, epoch_loss_df]
+                ).reset_index(drop=True)
+            else:
+                self.loss_values = epoch_loss_df
+
+            if self.verbose:
+                iterator.set_description(
+                    iterator_description.format(
+                        loss=loss.detach().cpu().item()))
```

Comment:
can we add an integration test that checks the value is printed in the right format?

Reply:
Sorry I meant an integration test without mocks that used something like [capsys](https://docs.pytest.org/en/7.4.x/how-to/capture-stdout-stderr.html#accessing-captured-output-from-a-test-function) to check the printed output